rlschool/liftsim/environment/env.py

```diff
@@ -63,7 +64,10 @@ def seed(self, seed=None):
         set_seed(seed)
 
     def step(self, action):
```

Comment:
You should check that the length of `action` is equal to `self._mansion.attribute.ElevatorNumber * 2`
`
action = np.array(action)
assert(action.shape[0]) == self._mansion.attribute.ElevatorNumber * 2, 'action is supposed to be an array of length 1.
`